### DISCUSSION TITLE
Differences in ODF due to use of specimen symmetry

### DISCUSSION THREAD

**Discussion by rohitmathew1987** [mtex-toolbox/mtex#770] [Ask Anything]
Hello all,

I seem to be finding differences in the plotted ODF for the same X-Ray texture data of a rolled FCC material. The code I use to plot the ODF is given below

-----------------------------------------------------------------------------------------------
%% Specify Crystal and Specimen Symmetries

% crystal symmetry
CS = crystalSymmetry('m-3m', [3.5 3.5 3.5], 'mineral', 'Ni');

% specimen symmetry
SS = specimenSymmetry('1');

% plotting convention
setMTEXpref('xAxisDirection','east');
setMTEXpref('zAxisDirection','outOfPlane');

%% Specify File Names

% path to files
pname = 'D:\Data\Bulk Texture\1';

% which files to be imported
fname = {...
  [pname '\111.xrdml'],...
  [pname '\200.xrdml'],...
  [pname '\220.xrdml'],...
  [pname '\113.xrdml'],...
  };

%% Specify Miller Indice

h = { ...
  Miller(1,1,1,CS),...
  Miller(1,0,0,CS),...
  Miller(1,1,0,CS),...
  Miller(1,1,3,CS),...
  };

%% Import the Data

% create a Pole Figure variable containing the data
pf = PoleFigure.load(fname,h,CS,SS,'interface','xrdml');

odf = calcODF(pf,'zero_range','resolution',5*degree,'silent');

%odf.SS = specimenSymmetry('222');

figure (1)
plot(odf,'phi2',[0 45 65]*degree,'contourf','minmax','silent')
mtexColorMap WhiteJet
colormap

----------------------------------------------------------------------------------------------------

I have used two ways to enforce orthotropic symmetry in the sample since it is rolled. When I initially use SS = specimenSymmetry('222') and **NOT** use odf.SS = specimenSymmetry('222'), I get the ODF (phi2 sections of 0,45 and 65) attached in **figure 1**. But if I use SS = specimenSymmetry('1') and use odf.SS = specimenSymmetry('222'), I get the ODF attached in **figure 2**. There is significant differences in the plotted ODFs and I am confused as to which would be the correct ODF since I would be estimating volume fractions of individual texture components.

I'd be very grateful for any help inunderstanding the error I'm making.

Regards,
Rohit

![Figure 1](https://user-images.githubusercontent.com/78981077/107814731-63b20600-6d98-11eb-8f07-c52c5c7e9cf7.jpg)
![Figure 2](https://user-images.githubusercontent.com/78981077/107814803-79bfc680-6d98-11eb-968c-c38a687845f6.jpg)

**Comment by ralfHielscher** [comment]
The first route is the correct one. Specimen symmetry should be specified before ODF computation.
Enforcing specimen symmetry by setting odf.SS = .... is not yet supported.

Ralf.

**Reply by rohitmathew1987** [reply]
Thank you Ralf for the quick reply. Although I can understand the code better, but the my doubt in the code came from the mtex documentation link "https://mtex-toolbox.github.io/EulerAngleSections.html" . The section - specimen symmetry, disucsses using odf.SS = specimenSymmetry('222') for obtaining the square ODFs with samples having orthhotropic symmetry.
